.github/workflows/functional-test.yaml

```diff
@@ -73,6 +73,7 @@ jobs:
     env:
       DE_IMAGE: 'ghcr.io/radius-project/deployment-engine'
       DE_TAG: 'latest'
+      FUNCTIONAL_TEST_APP_ID: 425843
```

Comment:
Assuming that this is a Test ID and will be removed?

Reply:
We need a GitHub app to use the Checks API.

Reply:
@AaronCrawfis has already configured a github app with this ID. We need to fix the private key in the repo secrets for this app (I had mistakenly given him a wrong value)